source/common/res/features/running-balance/main.js

```diff
@@ -108,12 +101,9 @@
       function onYnabGridyBodyChanged() {
         var accountController = ynabToolKit.shared.containerLookup('controller:accounts');
 
-        if (accountController.get('sortProperties').indexOf('date') !== -1) {
-          updateRunningBalanceCalculation(accountController.get('arrangedContent'));
-          updateRunningBalanceColumn();
-        } else {
-          $('.ynab-toolkit-grid-cell-running-balance').remove();
-        }
+        sortContent(accountController.get('content'));
```

Comment:
This might not work in about an hour. Previously Ember's `ArrayController` had some magic if you set the `model`, it would automatically create these properties with the SortableMixin. But that's now been deprecated. So the accounts controller is becoming just a regular controller and we are left to sort it on our own.

The sorted/filter results will be in the property `contentResults`.